>From 5cedc2655ff6cc0aa105eb6ce624932b3fbf311b Mon Sep 17 00:00:00 2001
From: Bob Ham <rah@settrans.net>
Date: Sun, 6 Nov 2016 13:16:07 +0000
Subject: [PATCH] include/meta.h: Fix bad template parameters in compiler macro

GCC 5.4 complains about the template parameters in MIN_VAL and
MAX_VAL which apparently need to be padded with space.

Signed-off-by: Bob Ham <rah@settrans.net>
---
 include/meta.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/meta.h b/include/meta.h
index 715620b..bcd30d0 100644
--- a/include/meta.h
+++ b/include/meta.h
@@ -222,8 +222,8 @@ private:
 
 #define POKE(x) for(;;) { ssize_t n=write(x, "", 1); if(n>0 || (EAGAIN!=errno && EINTR!=errno)) break;  }
 
-#define MIN_VAL(x) (std::numeric_limits<x>::min()) 
-#define MAX_VAL(x) (std::numeric_limits<x>::max()) 
+#define MIN_VAL(x) (std::numeric_limits< x >::min())
+#define MAX_VAL(x) (std::numeric_limits< x >::max())
 
 void appendLong(mstring &s, long val);
 
-- 
2.1.4

